.travis.yml

```diff
@@ -3,12 +3,13 @@ cache:
   directories:
     - WcaOnRails/vendor/bundle
 rvm:
-  - 2.3.1
+  - 2.4.0
```

Comment:
I know  `.ruby_version` also works in `rbenv`, because I'm now using it. 'rbenv' and 'RVM' should cover most developers.

For the rest we _could_ have a test asserting that `RUBY_VERSION` is equal to the contents of `.ruby_version`.